### PR TITLE
API | timothyHuxxSpec | Get Booking test suite

### PR DESCRIPTION
**Pull request by TimothyHuxx**

### CODE REVIEW

File: cypress/fixtures/apiData.json [hunk 1]
```diff
@@ -189,5 +189,29 @@
...
+                    "additionalneeds": "Breakfast"
+            },
+
+            "dataType": {
+                "firstname": "string",
+                "lastname": "string",
+                "totalprice": "number",
+                "depositpaid": "boolean",
+                "bookingdates": "object",
+                "additionalneeds": "string"
```
Comment (shabaiev) [inline cypress/fixtures/apiData.json:213]: why not a camel case?

**Review by derearte — APPROVED** [review]
Approved

File: cypress/fixtures/apiData.json (hunk 1 above)
Comment (shabaiev) [reply cypress/fixtures/apiData.json:213]: @TimothyHuxx

Comment (TimothyHuxx) [reply cypress/fixtures/apiData.json:213]: @shabaiev because in the response it's not camelCase

**Review by AnnaIurchyk — APPROVED** [review]

**Review by shabaiev — APPROVED** [review]

**Pull request merged**